RocketChat/Rocket.Chat
# [FIX] Preview message not showing when reply direct message

## Proposed changes (including videos or screenshots)





## Issue(s)


## Steps to test or reproduce


You should enable the `Hide Right Sidebar with Click` preference on the message section and reply a direct message. It is just driving you to the DM instead of adding the original message in a quote.

## Further comments



The on-click event is bubbling to the parent component when the `Hide Right Sidebar with Click` preference is enabled.
